test/HeadersTest.php

```diff
@@ -381,6 +381,7 @@ public function testHeaderCrLfAttackAddHeaderLineWithValue()
         $headers = new Mail\Headers();
         $this->setExpectedException('Zend\Mail\Header\Exception\InvalidArgumentException');
         $headers->addHeaderLine('Fake', "foo-bar\r\n\r\nevilContent");
+        $headers->forceLoading();
```

Comment:
headers are now really lazyloaded/lazyparsed, so this now throw exception when you acces them